collector-bitmovin-player/src/main/java/com/bitmovin/analytics/bitmovin/player/BitmovinSdkAdapter.java

```diff
@@ -206,6 +206,13 @@ public EventData createEventData() {
                     break;
             }
         }
+        // Override mpdUrl and m3u8 url if they are set in the AnalyticsConfig
```

Comment:
@Tigraine what should have precedence here? 
I think the config values should be the fallbacks, but if we can actually get that information from the player, we should use that?

@peterpf  depending on Daniel's answer, we could also always set this in the `EventData` constructor already and this can be independent of the SDK (so we don't have to do it in both). Or even if that's not the case, we could try to set those in one place, as it's duplicate code otherwise

Reply:
@rolandgriesser  Actually the order of precedence here is config > player.. Because the stuff we get from the player is actually wrong

Reply:
ah, I thought that's only for the cases where we don't get the information. Good to know - still should be done in the base collector IMO, as it's the same functionality, no matter the SDK

Reply:
yes - agree with the duplicate code.

But this is for RTL who are struggling with the streamRoot SDK setting 127.0.0.1 as MPD-URL so they want to be able to configure the "real" URL regardless of proxy.

Reply:
I was thinking about putting the code in the `sendEventData` in the analytics core, but was not sure if this is the right place. @rolandgriesser what do you think where the best place is to put this common code?

Reply:
Might need to refactor a bit here, as the `createEventData` from the adapter is also used in the ads - so putting this in `sendEventData` would not apply to ads. A way would be to have a `PlayerAdapter` base class that has a `createEventData` and an abstract `decorateEventData`, so we would create the eventData in the base class and the adapters would only decorate it then. 
@Tigraine thoughts?

Reply:
please do a pair programming session on this :)